openapi/components/schemas/Common/CommonBillingPortal.yaml

```
@@ -4,94 +4,108 @@ required:
   - slug
 properties:
   id:
-    description: The billing portal identifier string.
+    description: Billing portal identifier string.
     readOnly: true
     allOf:
       - $ref: ../ResourceId.yaml
   slug:
-    description: >-
-      The url path segment that will be appended to the billing portal
-      url (e.g. `https://rebilly.com/billing-portal/{slug}`) to make it human-readable and simple.
+    description: |-
+      Path segment that is appended to the billing portal URL to help
+      make it human-readable. Example: `https://company.com/billing-portal/{slug}`.
     type: string
     minLength: 5
     maxLength: 100
   websiteId:
-    description: Billing portal website ID.
+    description: |-
+      ID of the website that is associated with the billing portal. A
+      website is where your organization obtains a customer. It is
+      also the processor account and billing descriptor used for payment
```

```suggestion
      also the gateway account and billing descriptor used for payment
```
Is this what ou mean?